packages/commerce-sdk-react/src/hooks/ShopperOrders/cache.ts

```diff
@@ -5,16 +5,16 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {getCustomerBaskets} from '../ShopperCustomers/queryKeyHelpers'
-import {ApiClients, CacheUpdateMatrix, CacheUpdateUpdate, CacheUpdateInvalidate} from '../types'
+import {ApiClients, Argument, CacheUpdateMatrix, CacheUpdate, CacheUpdateUpdate, CacheUpdateInvalidate, MergedOptions} from '../types'
 import {getOrder} from './queryKeyHelpers'
 
 type Client = ApiClients['shopperOrders']
+/** Parameters that get passed around, includes client config and possible parameters from other endpoints */
+type GetOrderParameters = MergedOptions<Client, Argument<Client['getOrder']>>['parameters']
 
-/** Logs a warning to console (on startup) and returns nothing (method is unimplemented). */
-const TODO = (method: keyof Client) => {
-    console.warn(`Cache logic for '${method}' is not yet implemented.`)
-    return undefined
-}
+const invalidateOrderQuery = (customerId: string | null, {parameters}: {parameters: GetOrderParameters},): CacheUpdate => ({
```

Comment:
```suggestion
const invalidateOrderQuery = (customerId: string | null, {parameters}: GetOrderOptions): CacheUpdate => ({
```
[part 2] save a layer of nesting 😉 

(2 parts because GitHub hates the blank line in between)